packages/wonder-blocks-popover/src/components/popover.tsx

```diff
@@ -317,7 +333,14 @@ export default class Popover extends React.Component<Props, State> {
                             >
                                 {children}
                             </PopoverAnchor>
-                            {opened && this.renderPopper(uniqueId)}
+                            {disablePortal
+                                ? opened && this.renderPopper(uniqueId)
+                                : popperHost &&
+                                  opened &&
+                                  ReactDOM.createPortal(
+                                      this.renderPopper(uniqueId),
+                                      popperHost,
+                                  )}
```

Comment:
**suggestion:** We tend to use separate functions to avoid these kind of complex expressions. It would be ideal moving this to improve readability:

e.g.
```
maybeRenderPortal(uniqueId: string) {
  const {portal} = this.props;

  if (!opened) {
    return null;
  }

  // Attach the dialog to a Portal
  const popperHost = this.getHost();
  if (portal && popperHost) {
    return ReactDOM.createPortal(
      this.renderPopper(uniqueId),
      popperHost,
    );
  }

  // Otherwise, append the dialog next to the trigger element
  return this.renderPopper(uniqueId);
}